proto/mrt/mrt.c

```diff
@@ -739,6 +739,7 @@ mrt_table_dump_step(struct mrt_table_dump_state *s)
   s->bws = &bws;
 
   if (s->table_open)
+    s->bws->mp_reach = !s->ipv4;
```

Comment:
Since the above if statement doesn't have { } doesn't this mean we will always execute 734 now even if s->table_open is false. Is that intentional?

Reply:
Uhhh, nice catch Sandy! 🧐 

We've all been writing too much Python lately...

Reply:
I forgot to submit the final one